qtodotxt/lib/task_htmlizer.py

```diff
@@ -38,16 +38,43 @@ def task2html(self, task):
             html += ' <font color="gray">(created: {})</font>'.format(task.creation_date)
         return html
 
+    def _addUrl(self, word, color="none"):
+        if int(QtCore.QSettings().value("support_jira", 1)):
```

Comment:
the first patch was OK but not this. We cannot add special code for every systems on earth. if you thing special treatment is really necessary for some urls then we need to add some sort of plugin